src/oas3/accessors.ts

```diff
@@ -1,13 +1,20 @@
-import { get, pick, values } from 'lodash';
-import { OpenAPIObject, OperationObject, SecuritySchemeObject } from 'openapi3-ts';
-
-import { uniqFlatMap } from '../utils';
-
-export function getSecurities(
-  spec: Partial<OpenAPIObject>,
-  operation: Partial<OperationObject>,
-): SecuritySchemeObject[] {
-  const globalSchemes = uniqFlatMap(spec.security);
-  const operationSchemes = uniqFlatMap(operation.security);
-  return values(pick(get(spec, 'components.securitySchemes'), operation.security ? operationSchemes : globalSchemes));
+import { get } from 'lodash';
+import { OpenAPIObject, OperationObject } from 'openapi3-ts';
+
+import { mapToKeys } from '../utils';
+
+export function getSecurities(spec: Partial<OpenAPIObject>, operation: Partial<OperationObject>): any {
```

Comment:
Did you mean?

```suggestion
export function getSecurities(spec: Partial<OpenAPIObject>, operation: Partial<OperationObject>): SecuritySchemeObject[][] {
```

Shouldn't be `any`